data/json/items/gunmod/grip.json

```diff
@@ -3,6 +3,7 @@
     "id": "light_grip",
     "type": "GUNMOD",
     "name": "lightweight replacement furniture",
+    "name_plural": "lightweight replacement furniture",
```

Comment:
"Lightweight replacement furnitures" was the correct plural.  Since it refers to a singular item, furniture loses it's uncountablility.